testdata/QDESQ/QDESQ_Sample_preparations.tsv

```diff
@@ -0,0 +1,9 @@
+QBiC Code	Secondary Name	Lab ID	Sample Type	Source	Source Name(s)	Source Lab ID(s)	Extract Code(s)	Condition: cellline
```

Comment:
Could you remove here the columns that do not have to do with this QDESQ project?